src/auth/auth.service.ts

```diff
@@ -372,4 +374,70 @@ export class AuthService {
       return internalServerError();
     }
   }
+
+  async deleteUserByUserId(
+    userId: number,
+    id: number,
+  ): Promise<void | CustomException> {
+    if (userId !== id) {
+      return forbidden();
+    }
+
+    const user = await this.prisma.user.findUnique({
+      where: {
+        id,
+        isDeleted: false,
+      },
+    });
+    if (!user) {
+      return notFound();
+    }
+
+    const socialType = user.socialType;
+
+    switch (socialType) {
+      case 1:
```

Comment:
```suggestion
    switch (SOCIAL_TYPE[user.socialType]) {
      case 'kakao':
```

`common/objects/social.ts` 에 있는 object 로 value 를 통해 비교한다면 1, 2 가 아니라 'kakao', 'apple' 로 비교 할 수 있어서 더 가독성이 좋을 것 같습니다.

Reply:
굿 저도 헷갈렸는데 일케 수정해볼게요